enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,252 @@
+reviewers:
+  - "@staebler"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2021-05-13
+last-updated: 2021-05-13
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CORS-1650"
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# RHEL 8 Server Worker/Infra Node Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add RHEL 8 Server support for deploying worker/infra
+nodes with [openshift-ansible] RHEL 8 nodes will be deployed and upgraded using
+the same Ansible playbook process however, RPM content will be delivered using
+an image in the release package.
+
+## Motivation
+
+Customers desire to deploy RHEL 8 nodes and using containerized content will provide
+a better user experience than configuring RPM repositories on nodes.  Using the
+existing process will allow near-term support of RHEL 8 nodes while enhanced
+longer-term RHEL 8 support is developed.
+
+### Goals
+
+1. Implement an os-content image for RHEL 8 RPMs
+1. Update [openshift-ansible] scaleup.yml and upgrade.yml playbooks to support RHEL 8 workers
+
+### Non-Goals
+
+* Deploying/upgrading RHEL 8 worker nodes through other mechanisms, i.e. operators
+* Changing the deployment/upgrade process for RHEL 7 worker nodes
+* Upgrading existing RHEL 7 worker nodes to RHEL 8
+
+## Proposal
+
+Currently, for RHEL 7 worker node deployment, the user must [prepare the host][prepare-rhel]
+before running the scaleup.yml playbook.  This process can be cumbersome for
+automating deployment and also requires additional steps to update repos prior
+to performing an upgrade.  For RHEL 8 hosts preparing repos on the hosts will
+not be required because the necessary packages will be delivered as an image in
+the release package.  This removes the need of mirroring the release package as
+well as setting up repo mirrors for offline or disconnected deployments.
+
+The basic process for scaling up and upgrading RHEL worker nodes will remain the
+same.  The playbooks will be updated to download and mount the image with the RPM
+content during the install/upgrade for RHEL 8 hosts.  The install process will
+not change for RHEL 7 worker nodes.
+
+[prepare-rhel]: https://docs.openshift.com/container-platform/4.7/machine_management/adding-rhel-compute.html#rhel-preparing-node_adding-rhel-compute
+
+### User Stories
+
+1. As an OpenShift admin using [openshift-ansible], I want to deploy and upgrade
+   RHEL 8 Server worker nodes without needing to prepare hosts for installation.
+
+1. As an Openshift admin, I want to be able to deploy RHEL 8 workers without
+   having to mirror RPM content.
```

Comment:
Should we just integrate better with Satellite of this?

Reply:
We need to better understand the customer's pain points with repo management.

Reply:
I don't think we want to talk about Satellite, avoiding the OCP repo management is a specific goal of this enhancement and given they've already got to mirror the release image it seems like we shouldn't concern ourselves with that in this enhancement.